swagger.yaml

```diff
@@ -593,6 +581,7 @@ definitions:
       "@graph":
         type: array
         items:
+          type: object
```

Comment:
this was what allowed `swagger-codegen` to get past the fatal error that kept it from outputting client code.